visualization/app/codeCharta/assets/scenarios.json

```diff
@@ -30,9 +26,7 @@
   {
     "name": "Coverage",
     "settings": {
-      "appSettings": {
```

Comment:
@BridgeAR I am not sure if this setting had some effect. I compared the default `codecharta.cc.json` and could see no difference

Reply:
This is applied while choosing the scenarios. I guess it does indeed change the behavior a bit but I did not yet run the code to verify it.

Reply:
If it does change something, I would appreciate a little demonstration :)